src/alvarium/annotators/handler/contracts.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from enum import Enum
+
+HTTP_REQUEST_KEY = "HttpRequestKey"
+CONTENT_TYPE = "Content-Type"
+CONTENT_LENGTH = "Content-Length"
+class DerivedComponent(Enum):
+    Method = "@method"
+    TargetURI = "@target-uri"
+    Authority = "@authority"
+    Scheme = "@scheme"
+    Path = "@path"
+    Query = "@query"
+    QueryParams = "@query-params"
+
+    def __str__(self) -> str:
+        return f'{self.value}'
+@dataclass
+class parseResult:
```

Comment:
classes should follow the PascalCase naming convention. Also, I believe there's a typo in the name of this class, it should be: `ParsedResult`.